### ISSUE TITLE
Vasp stresses are confusing and unit is a mystery

### ISSUE THREAD

**Issue by Leimeroth** [pyiron/pyiron_atomistics#1071]
Lammps and vasp stores pressures/stresses somehow different and vasp stresses are confusing.
Default lammps jobs store pressures as full matrix and stress =-pressure.
For vasp the output contains a 3 vector pressure and a matrix stresses,
but the stresses do not give any reasonable result when trying to caculate elastic constants with them and a short look
in the code made me assume that units are not the same as for pressures.

**Comment by pmrv** [comment]
Yes, the current value of `pressures` makes no sense, I introduced `stresses` for backwards-compat, but it's been a while and imo we can throw the old `pressures` out.  I'm not sure about the unit off the top my head, but I've used the stress output before successfully.

**Comment by Leimeroth** [comment]
To get stresses from the format written in hdf to GPa (which is the pyiron unit for pressure, so it makes sense for stresses too imo) I need to  stress = stress/KBAR_TO_EVA*0.1
